package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "demo-passkey-wallet",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
is this intentional? I don't think we should have a package-lock.json file at the root of the repo. Only in the `frontend/` folder.

Reply:
huh that's so weird and not intentional. i reproduced it by running `npm install` in the root dir and it errored but created this file. deleting it now, thanks!